backend/src/services/mempool_monitor.py

```
@@ -167,53 +179,33 @@ async def _process_rugpull_indicators(self, signature: str, logs: list):
-                            if self.socketio:
-                                self.socketio.emit('rugpull_alert', {
-                                    'signature': signature,
-                                    'reason': 'Token account closed',
-                                    'details': {'account': str(account_to_close)}
-                                })
-
-                    except Exception as parse_e:
-                        logger.warning(f"Error parsing token instruction for rugpull check: {parse_e}")
+                if str(instruction.program_id) == str(TOKEN_PROGRAM_ID):
+                    data = instruction.data
+                    if not data: continue
```

Ruff (0.15.4)

[error] Multiple statements on one line (colon)

(E701)